src/mono/wasm/runtime/lazyLoading.ts

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable no-prototype-builtins */
```

Comment:
`no-prototype-builtins`, why ?

Reply:
It was complaining about `hasOwnProperty`

Reply:
Is that stupid rule and should be disabled in the whole project ? Or there is idiomatic way how this should be done?

Reply:
`Object.prototype.hasOwnProperty.call(foo, "bar")` https://eslint.org/docs/latest/rules/no-prototype-builtins

I'll update the code.